ipatests/test_integration/test_trust.py

```diff
@@ -268,6 +253,69 @@ def test_override_homedir(self):
             tasks.restore_files(self.master)
             tasks.clear_sssd_cache(self.master)
 
+    def test_extdom_plugin(self):
+        """Extdom plugin should not return error (32)/'No such object'
+
+        Regression test for https://pagure.io/freeipa/issue/8044
+
+        If there is a timeout during a request to SSSD the extdom plugin
+        should not return error 'No such object' and the existing user should
+        not be added to negative cache on the client.
+        """
+        search_dn = DN(
```

Comment:
Please try to avoid generic names like this one. Try to name variables according to their contents or function.
In this particular case it can be called "extdom_config_dn"